src/main/scala/com/fulcrumgenomics/bam/ClipBam.scala

```diff
@@ -123,7 +128,7 @@ class ClipBam
     this.clipper.clip5PrimeEndOfRead(r2, readTwoFivePrime)
     this.clipper.clip3PrimeEndOfRead(r2, readTwoThreePrime)
 
-    if (r1.unmapped || r2.unmapped || r1.refIndex != r2.refIndex || r1.pairOrientation != PairOrientation.FR) {
+    if (!overlappingReads || r1.unmapped || r2.unmapped || r1.refIndex != r2.refIndex || r1.pairOrientation != PairOrientation.FR) {
```

Comment:
I find this a little confusing.  How about `if (overlappingReads && r1.mapped && r2.mapped $$ r1.refIndex == r2.refIndex && r1.pairOrientation == FR) { ... }`.  I'm assuming there used to be something more than `// do nothing` in the first branch, but now there's not there's no reason to even have it.

Also, the ship has probably sailed, but `overlappingReads` is confusing because I interpreted it initially as "reads are overlapping" not "clip overlapping reads".